tests/unit/test_dash.py

```diff
@@ -109,3 +112,70 @@ def test_kurtosis():
     kui = str(round(data['pnl'].kurtosis(), 2))
```

Comment:
why 2? what is 2?

Reply:
round floating points to only keep 2 decimals

Reply:
this is fine. thanks